pkg/cmd/roachtest/tests/cluster_to_cluster.go

```diff
@@ -213,18 +213,29 @@ func defaultWorkloadDriver(
 }
 
 type replicateTPCC struct {
-	warehouses int
+	warehouses     int
+	duration       time.Duration
+	repairOrderIDs bool
 }
 
 func (tpcc replicateTPCC) sourceInitCmd(tenantName string, nodes option.NodeListOption) string {
-	return fmt.Sprintf(`./cockroach workload init tpcc --data-loader import --warehouses %d {pgurl%s:%s}`,
-		tpcc.warehouses, nodes, tenantName)
+	cmd := roachtestutil.NewCommand(`./cockroach workload init tpcc`).
+		Flag("data-loader", "import").
+		Flag("warehouses", tpcc.warehouses).
+		Arg("{pgurl%s:%s}", nodes, tenantName)
+	return cmd.String()
 }
 
 func (tpcc replicateTPCC) sourceRunCmd(tenantName string, nodes option.NodeListOption) string {
 	// added --tolerate-errors flags to prevent test from flaking due to a transaction retry error
-	return fmt.Sprintf(`./cockroach workload run tpcc --warehouses %d --tolerate-errors {pgurl%s:%s}`,
-		tpcc.warehouses, nodes, tenantName)
+	cmd := roachtestutil.NewCommand(`./cockroach workload run tpcc`).
+		Flag("warehouses", tpcc.warehouses).
+		MaybeFlag(tpcc.duration > 0, "duration", tpcc.duration).
+		Option("tolerate-errors").
```

Comment:
Hm.  I feel like tolerate-errors substantially reduces the correctness checking value of the test. Did we dig into why we thought we needed it here at all? Can't the workload retry retriable errors without it (perhaps with a different flag)?

Reply:
Done.